e2e/iam/org_api_key_access_list_test.go

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//go:build e2e || (iam && atlas) || (iam && om50) || (!om44 && !opsmanager)
+//go:build (e2e || iam || (iam && om50)) && !om44 && !opsmanager
```

Comment:
could we drop `om50` given this should cover eventually om60?
```suggestion
//go:build (e2e || iam) && !om44 && !opsmanager
```

Reply:
also what's the use of `!opsmanager` wouldn't that mean never for ops manager or something like it? that's how I see it

Reply:
Do you want to keep om44 and drop only om50? if yes, I was thinking to use opsmanager for 50. is this fine? are you proposing something different?

Reply:
`ops_manager_4_4_generic_e2e` use `opsmanager` as a tag so` !opsmanager` was just preventing running this test as part of that evergreen task. I believe I can remove it with this new build formula

Reply:
Updated

Reply:
original comment is about going disallow list only rather than allow and disallow at the same time, if the test calls `iam` it runs all of iam, `iam,om44` runs all iam except for those with `!om44` indicating not supported, and so on so we don't need to revisit the tags when a new version of OM comes next year

The use of `opsmanager` is honest question, not sure how it's been used here

Reply:
with the latest changes, I am using `opsmanger` as the latest version, in this case 5.0. In the future, `opsmanager` would represent 6.0

Reply:
with the latest changes, I am using `opsmanger` as the latest version, in this case 5.0. In the future, `opsmanager` would represent 6.0

Reply:
what will happen when a feature is `om60` only, you'll need to add a `!om50` to that test meaning you now need to change the test for ops manger 5.0 to be `om50` instead of `opsmanager`, this is what I'm trying to prevent next year and why I'm saying go disallow list, by default all test run, unless not allowed to run